src/relay_sdk/events.py

```diff
@@ -13,18 +13,27 @@ class Events:
     def __init__(self, client: Session) -> None:
         self._client = client
 
-    def emit(self, data: Mapping[str, Any]) -> None:
-        """Create and send an event to the service.
+    def emit(self, data: Mapping[str, Any], key: Any = None) -> None:
```

Comment:
You can express a `None`-able type using `Optional` (just import it from `typing`):

```python
def emit(..., key: Optional[str] = None) -> ...:
```

Reply:
Thanks, I'll work on cleaning this all up a bit today. So far I've basically just managed to get it working ...
This is exactly my second foray into Python; let's keep that between ourselves though, eh?